src/psyclone/generator.py

```diff
@@ -440,13 +440,7 @@ def main(args):
     except Exception:  # pylint: disable=broad-except
         print("Error, unexpected exception, please report to the authors:",
               file=sys.stderr)
```

Comment:
There's a TODO for #753 in this file but that issue has been closed. Please update.

Reply:
Well spotted. I've had to create a new issue as I'd not captured this.